app/db-migrate

```diff
@@ -7,6 +7,7 @@ psql \
   --port=$DB_PORT \
   --username=$DB_USER \
   --dbname=$DB_NAME \
-  --variable=ON_ERROR_STOP=1 \
+  --set=SEARCH_PATH=$DB_SCHEMA \
+  --set=ON_ERROR_STOP=1 \
```

Comment:
Curious about the change in syntax from `variable=` to `set=`?

Reply:
I actually have zero preference between the two. I think the first time I wrote it I thought `variable` was clearer, but I think the second time I came around to add the extra variable I didn't actually realize that `variable` and `set` meant the same thing, and the example I referenced used `set` so 🤷 

Do you have a preference and want to be the tie breaker?

Reply:
No preference, just wanted to make sure I wasn't missing anything!